js/info.js

```diff
@@ -754,6 +772,8 @@ var info = (function () {
             _panelsTemplate["bottom-panel"] = configuration.getConfiguration().application.templatebottominfopanel;
         }
         _sourceOverlay = mviewer.getSourceOverlay();
+        _sourceSelectOverlay = mviewer.getSourceSelectOverlay();
+        _sourceSubSelectOverlay = mviewer.getSourceSubSelectOverlay();
```

Comment:
When do you reuse `_sourceSelectOverlay ` and `_sourceSubSelectOverlay ` from `info.js` file ?

Reply:
Good question! Thanks for your attentive review!